libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushMessaging.kt

```diff
@@ -41,7 +41,7 @@ import com.salesforce.androidsdk.config.BootConfig
 import com.salesforce.androidsdk.push.PushNotificationsRegistrationChangeWorker.PushNotificationsRegistrationAction
 import com.salesforce.androidsdk.push.PushNotificationsRegistrationChangeWorker.PushNotificationsRegistrationAction.Deregister
 import com.salesforce.androidsdk.push.PushNotificationsRegistrationChangeWorker.PushNotificationsRegistrationAction.Register
-import com.salesforce.androidsdk.push.PushService.enqueuePushNotificationsRegistrationWork
+import com.salesforce.androidsdk.push.PushService.Companion.enqueuePushNotificationsRegistrationWork
```

Comment:
Nice, love getting rid of static methods.  And since `enqueuePushNotificationsRegistrationWork` was originally package private in Java and now internal in Kotlin so  this isn't a breaking change.